sdk/workloads/arm-workloadssapvirtualinstance/review/arm-workloadssapvirtualinstance.api.md

```diff
@@ -1323,11 +1077,11 @@ export interface SingleServerConfiguration extends InfrastructureConfiguration {
 
 // @public
 export interface SingleServerCustomResourceNames {
```

Comment:
@kazrael2119 could you find spec for me?

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/workloads/Workloads.SAPVirtualInstance.Management/models.tsp#L1861